docs/install/agent_install_syntax.rst

```diff
@@ -72,4 +92,9 @@ Non-interactive install syntax
   
   .. note:: By default the DCM Agent is configured to not be started at system boot.  If you wish to have the DCM Agent configured to be started at system boot then specify the **-B** or **--on-boot** option.
 
+  .. note:: By passing the **--intrusion-detection-ossec** flag, the agent will download and install ossec, and will process alerts
+            from it that are level 5 and above.  Any alerts below 10 are only stored in the agent database.  Any alerts
```

Comment:
Can you add some text saying something like "the agent will download and install ossec via your Linux distribution system packages"  just to have it in the doc that we are respecting the license.